internal/commands/initialize/initialize.go

```diff
@@ -67,28 +77,130 @@ func NewCmdInit() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&o.Template, "template", "", "set template (url or internal)")
-	cmd.Flags().StringVar(&o.Output, "output", "", "set output dir")
 	cmd.Flags().BoolVar(&o.ShowList, "list", false, "show list of examples")
 
 	return cmd
 }
 
-func (o *InitOptions) Validate(cmd *cobra.Command) error {
-	if len(o.Template) == 0 {
-		cmd.Help()
-		return fmt.Errorf("template must be specified\n")
+func (o *InitOptions) Complete(cmd *cobra.Command) error {
+	o.Output = "."
+	if len(cmd.Flags().Args()) != 0 {
+		o.Output = cmd.Flags().Args()[0]
 	}
 
 	return nil
 }
 
+func (o *InitOptions) Validate(cmd *cobra.Command) error {
+	return nil
+}
+
 func (o *InitOptions) Run() error {
-	dest, err := generate.GenerateFiles(o.Template, o.Output)
+	if len(o.Template) != 0 {
+		dest, err := generate.GenerateFiles(o.Template, o.Output)
+		if err != nil {
+			return err
+		}
+
+		pterm.Success.Printfln(`Initialized project from template "%s" to %s`, o.Template, dest)
+
+		return nil
+	}
+
+	namespace := ""
+	envList := []string{}
+
+	env := os.Getenv("ENV")
+	if len(env) == 0 {
+		env = "dev"
+	}
+
+	dir := o.Output
+	if dir == "." {
+		cwd, err := os.Getwd()
+		if err != nil {
+			return fmt.Errorf("can't get current work directory: %s", cwd)
+		}
+	}
+
+	dir, err := filepath.Abs(o.Output)
+	if err != nil {
+		return fmt.Errorf("can't init: %w", err)
+	}
+
+	fmt.Println(dir)
+
+	namespace = filepath.Base(dir)
+	err = survey.AskOne(
+		&survey.Input{
+			Message: fmt.Sprintf("Which namespace will you have?"),
+			Default: namespace,
+		},
+		&namespace,
+		survey.WithValidator(survey.Required),
+	)
 	if err != nil {
-		return err
+		return fmt.Errorf("can't init: %w", err)
 	}
 
-	pterm.Success.Printfln(`Initialized project from template "%s" to %s`, o.Template, dest)
+	err = survey.AskOne(
+		&survey.Input{
+			Message: fmt.Sprintf("Which environments will you have?"),
+			Default: env,
+		},
+		&env,
+		survey.WithValidator(survey.Required),
+	)
+	if err != nil {
+		return fmt.Errorf("can't init: %w", err)
+	}
+
+	envList = append(envList, env)
+	env = ""
+
+	for {
+		err = survey.AskOne(
+			&survey.Input{
+				Message: fmt.Sprintf("Another environment? [enter - skip]"),
+				Default: env,
+			},
+			&env,
+		)
+		if err != nil {
+			return fmt.Errorf("can't init: %w", err)
+		}
+
+		if env == "" {
+			break
+		}
+
+		envList = append(envList, env)
+		env = ""
+	}
+
+	for _, v := range envList {
+		envPath := filepath.Join(dir, ".ize", "env", v)
+		err := os.MkdirAll(envPath, 0755)
+		if err != nil {
+			return fmt.Errorf("can't create dir by path %s: %w", envPath, err)
+		}
+
+		viper.Reset()
+		cfg := make(map[string]string)
+		cfg["namespace"] = namespace
+		cfg["env"] = v
+
+		raw := make(map[string]interface{}, len(cfg))
+		for k, v := range cfg {
+			raw[k] = v
+		}
+
+		viper.MergeConfigMap(raw)
+		err = viper.WriteConfigAs(filepath.Join(envPath, "ize.toml"))
```

Comment:
I'm wondering why viper writes configs as literal strings (single quotes) and not regular strings (double quotes).
I would prefer to use double quotes for normal values, do you know if that would be possible @psihachina ?